scripts/build.sh

```diff
@@ -4,7 +4,8 @@ set -ex
 SCRIPT_DIR=$(cd $(dirname "$0") && pwd -P)
 ROOT=$(dirname $SCRIPT_DIR)
 
-if [ "$1" = "--install" ]; then
+if ! [ -x "$(command -v bikeshed)" ] || [ "$1" = "--upgrade" ]; then
```

Comment:
I don't love installing software by default, but I guess the next easiest option involves writing a proper python script or something, so this is probably fine.

Reply:
I'm happy to just land this for now, but generally it seems better not to write outside the project without at least getting am opt-in.